### PR TITLE
Setup webpack

### PR DESCRIPTION
**Pull request by Elijahdre**
- I setup webpack
- I added Linters file

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (niicode) [inline src/index.html:13]: You should add something to the body of your html file eg. `<p>Hello world</p>`. This will help pass all the linters

**Review by niicode — CHANGES_REQUESTED** [review]
Hi @Elijahdre , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great project implementation :heavy_check_mark: 
- Good use of arrow function :heavy_check_mark: 
- Clean code :heavy_check_mark: 
- App works as expected :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@niicode ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Elijahdre) [reply src/index.html:13]: @niicode Thank you for the feedback

**Review by niicode — APPROVED** [review]
Hi @Elijahdre , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**